parsers/verilog.c

```diff
@@ -247,7 +252,8 @@ static const keywordAssoc KeywordTable [] = {
 	{ "typedef",       	K_TYPEDEF,   	{ 1, 0 } },
 	{ "union",         	K_STRUCT,    	{ 1, 0 } },
 	{ "var",           	K_REGISTER,  	{ 1, 0 } },
-	{ "void",          	K_REGISTER,  	{ 1, 0 } }
+	{ "void",          	K_REGISTER,  	{ 1, 0 } },
+	{ "with",          	K_WITH,			{ 1, 0 } }
```

Comment:
(just comment) I recommend adding "," at the end. So we can make diff smaller when you add a new keyword next time.

Reply:
I know modern languages allow this, but I have been misunderstanding that C did not allowed this.

I've checked the K&R both the 1st and the 2nd edition, Most of examples do not use semicolon after the last item, but syntax rule says;

```
initializer:
	assignment-expression
	{ initializer-list }
	{ initializer-list , }


initializer-list:
	initializer
	initializer-list , initializer
```

Verilog/SystemVerilog does not allow this, it makes difficult scripting...

Thank!!!